packages/mrm-core/src/__tests__/npm.spec.js

```diff
@@ -231,6 +231,43 @@ describe('install()', () => {
 		expect(fn).not.toThrow('Invalid npm version');
 	});
 
+	it('should not automatically add version data to non-registry installs', () => {
```

Comment:
Let's use `it.each` here:

https://github.com/sapegin/jest-cheat-sheet/blob/master/Readme.md#data-driven-tests-jest-23